safe-global/web-core
# fix: don't show previous Safe notifications

## What it solves

Resolves https://github.com/safe-global/web-core/issues/1267

## How this PR fixes it
Before checking if the Safe version is outdated, verifies that the loaded Safe is the selected one.

## How to test it
1. Open a 1.1.1 Safe, which should show an update notification
2. Change to a > 1.3.0 Safe and there should be no `Update Safe` notification popping up or in the Notification center

## Screenshots
![outdated notifications fix](https://user-images.githubusercontent.com/32431609/210233810-6b2b1915-fd80-431f-b471-5f99998c6324.gif)
